### ISSUE TITLE
Regression in v0.14.9 for SVG with text node

### ISSUE THREAD

**Issue by capnm** [thorvg/thorvg#2715]
**This is an update regarding a similar regression in v0.13.8 that fixed (or possibly reintroduced) part of it.**

- It works as intended 
· in the official Godot 4.4dev1,  **test1.svg, test2.svg** render the yellow rectangle.
·  in my static TVG build...

- I can reproduce it now with **test1.svg** in
· [TVG viewer](https://www.thorvg.org/viewer) (empty)
 · Godot 4.4 dev/master + TVG v0.14.9 build (error)

MRP test files [mrp-svg_text-tests-godot.zip](https://github.com/user-attachments/files/16909823/mrp-svg_text-tests-godot.zip)

- text1.svg
```xml
<svg
   width="256"
   height="128"
   id="svg_text_bug"
   xmlns="http://www.w3.org/2000/svg"
>

 <rect width="100" height="100" fill="yellow" />
 <text x="50" y="50">m</text>

</svg>
```

![text1](https://github.com/user-attachments/assets/06ead697-6f07-4e2b-9e4e-d5be2237f2af)

- text2.svg
```xml
<svg
   width="256"
   height="128"
   id="svg_text_ok"
   xmlns="http://www.w3.org/2000/svg"
>

 <rect width="100" height="100" fill="yellow" />
 <text x="50" y="50"></text>

</svg>
```
![text2](https://github.com/user-attachments/assets/27db3981-1bdb-4764-a1dc-99e1abcad01b)

<details><summary>Screenshot</summary>
<p>

![image](https://github.com/user-attachments/assets/391e5242-6a7f-4726-b573-19d611f2f1da)

</p>
</details>

**Comment by hermet** [comment]
@capnm This issue may occur due to how Godot strictly handles error handling from the thorvg APIs. A fix will be included in v0.14.10. Please reopen the issue if it persists with Godot after the update. Thank you.

**Comment by capnm** [comment]
@hermet Perfect catch! Just a single line fixes all the issues :-)
I could try to commit v0.14.9 together with the patch.

By the way, could you double-check what to do with this one (in v0.14.9)? I made a mistake here once before :)
https://github.com/godotengine/godot/tree/master/thirdparty/thorvg/patches
pr2702-sw_engine-handle-small-cubics.patch

Thanks!
